src/traveler.js

```diff
@@ -0,0 +1,57 @@
+import Trip from '../src/trip.js';
+
+class Traveler {
+  constructor(travelerData) {
+    this.id = travelerData.id
+    this.name = travelerData.name
+    this.travelerType = travelerData.travelerType
+    this.allTrips = []
+    this.pastTrips = []
+    this.futureTrips = []
+    this.pendingTrips = []
+    this.currentLocation = "Denver, Colorado"
+  }
+  sayFirstName() {
```

Comment:
I like that you thought to add `sayFirstName()` assuming you're considering what you'd like to do with the `DOM`, great idea!